bin/main.rb

```diff
@@ -0,0 +1,82 @@
+require_relative '../lib/game.rb'
+require_relative '../lib/player.rb'
+require_relative '../lib/board.rb'
+
+# welcome info
+
+puts
+puts "*" * 27
+puts "  WELCOME TO TIC-TAC-TOE!"
+puts "*" * 27
+puts
+
+puts 'What is your name player 1?'
+name_1 = gets.chomp
+
+puts 'What is your name player 2?'
+name_2 = gets.chomp
+
+#initialization
+
+player1 = Player.new(name_1, 1)
+player2 = Player.new(name_2, 2)
+board = Board.new
+game = Game.new(player1.name, player2.name)
+
+
+#game
+
+puts 'Let\'s play!'
+
+count = 0
+
+while count < 9
+
+  puts board.display_board
+
+  if count.even?
+
+    puts 'Choose the number where you want to place your move'
+    move_player1 = gets.chomp
+
+    if board.set_index(move_player1.to_i, player1.player_number)
+      puts 'Good move!'
+      count += 1
+    else
+      puts 'Invalid move, try again!'
+    end
+
```

Comment:
I suggest you create a method in the Game class to check if the move is ok. That method will call a method from `Board` class

For example
```ruby
def move_valid?
    ...
    if board.valid input

    end
    ...
end
```